src/Widgets/ImageWidget/ImageWidgetComponent.js

```diff
@@ -1,26 +1,42 @@
 import * as React from "react";
 import * as Scrivito from "scrivito";
+import AOS from "aos";
 
-Scrivito.provideComponent("ImageWidget", ({ widget }) => {
-  let image = (
-    <Scrivito.ImageTag
-      content={widget}
-      attribute="image"
-      alt={alternativeText(widget)}
-    />
-  );
-
-  const link = widget.get("link");
-  if (link && !Scrivito.isInPlaceEditingActive()) {
-    image = <Scrivito.LinkTag to={link}>{image}</Scrivito.LinkTag>;
+class ImageWidget extends React.Component {
+  constructor(props, context) {
+    super(props, context);
+    AOS.init();
   }
 
-  if (["center", "right"].includes(widget.get("alignment"))) {
-    return <div className={`text-${widget.get("alignment")}`}>{image}</div>;
+  getDerivedStateFromProps() {
```

Comment:
Please do not use [`getDerivedStateFromProps`](https://reactjs.org/docs/react-component.html#static-getderivedstatefromprops) for this use-case. To quote the documentation:

> [`getDerivedStateFromProps`] exists for [**rare use cases**](https://reactjs.org/blog/2018/06/07/you-probably-dont-need-derived-state.html#when-to-use-derived-state) where the state depends on changes in props over time. 

In this case a [`componentDidUpdate`](https://reactjs.org/docs/react-component.html#componentdidupdate) is probably better suited for this use-case.